README.md

```diff
@@ -51,49 +56,42 @@ Only `-manufacturer` is required, everything else is optional. For example if
 omitted, no email is sent at the end of the setup process. Use `efe -help` to
 see all the options.
 
-
 ## Manual SDCard selection
 
 If your workstation has more than one removable disk, it will not select one
 automatically and will ask you to specify one. You have to specify it with
 `-sdcard`:
 
-- Linux: it is in the form of `/dev/sdX` or `/dev/mmcblkN`.
-- OSX: It is in the form of `/dev/diskX`. You can identify the disk of your
-  SDCard by running: `diskutil list`.  It will look like `/dev/disk2`.
-
+* Linux: it is in the form of `/dev/sdX` or `/dev/mmcblkN`.
```

Comment:
Not a fan of the churn, since the whole site uses this style. I'm not particularly attached to it but at least it's consistent.